apps/meteor/server/models/raw/Users.js

```diff
@@ -3079,4 +3079,17 @@ export class UsersRaw extends BaseRaw {
 			},
 		);
 	}
+
+	countActiveUsersWithCustomRoles() {
+		const customRoleRegex = /^[0-9a-fA-F]{24}$/;
+		const query = {
+			active: true,
```

Comment:
This query only considers the active field but for a user to be truly active it has to have both active true and the lastLoggin field, otherwise, the stats would take pending users into consideration and this could possibly skew the statistics.

Reply:
Thanks for the feedback @rique223 , I'm checking with product team